FTBFS with GCC 13

Fails with:

```
[   78s] cd /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/build/libdnf && /usr/bin/c++ -DLIBDNF_UNSTABLE_API -D_FILE_OFFSET_BITS=64 -Dlibdnf_EXPORTS -I/usr/include/gio-unix-2.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/json-c -I/usr/include/libxml2 -I/home/abuild/rpmbuild/BUILD/libdnf-0.67.0 -I/home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/utils -I/home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/transaction -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -DWITH_ZCHUNK -std=c++11 -Wmissing-declarations -O2 -g -DNDEBUG -fPIC   -DPACKAGE_VERSION=\"0.67.0\" -DGETTEXT_DOMAIN=\"libdnf\" -DG_LOG_DOMAIN=\"libdnf\" -DTESTDATADIR=\"/home/abuild/rpmbuild/BUILD/libdnf-0.67.0/data/tests\" -Wcast-align -Wno-uninitialized -Wredundant-decls -Wwrite-strings -Wformat-nonliteral -Wmissing-format-attribute -Wsign-compare -Wtype-limits -Wuninitialized -Wall -Wl,--as-needed -MD -MT libdnf/CMakeFiles/libdnf.dir/dnf-advisoryref.cpp.o -MF CMakeFiles/libdnf.dir/dnf-advisoryref.cpp.o.d -o CMakeFiles/libdnf.dir/dnf-advisoryref.cpp.o -c /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/dnf-advisoryref.cpp
[   78s] In file included from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/Swdb.hpp:38,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/query.hpp:32,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/dnf-sack-private.hpp:31,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/dnf-advisoryref.cpp:40:
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/../utils/sqlite3/Sqlite3.hpp:100:33: error: 'std::int64_t' has not been declared
[   78s]   100 |         void bind(int pos, std::int64_t val)
[   78s]       |                                 ^~~~~~~
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/../utils/sqlite3/Sqlite3.hpp:100:14: error: 'void SQLite3::Statement::bind(int, int)' cannot be overloaded with 'void SQLite3::Statement::bind(int, int)'
[   78s]   100 |         void bind(int pos, std::int64_t val)
[   78s]       |              ^~~~
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/../utils/sqlite3/Sqlite3.hpp:93:14: note: previous declaration 'void SQLite3::Statement::bind(int, int)'
[   78s]    93 |         void bind(int pos, int val)
[   78s]       |              ^~~~
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/../utils/sqlite3/Sqlite3.hpp:107:33: error: 'std::uint32_t' has not been declared
[   78s]   107 |         void bind(int pos, std::uint32_t val)
[   78s]       |                                 ^~~~~~~~
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/../utils/sqlite3/Sqlite3.hpp:107:14: error: 'void SQLite3::Statement::bind(int, int)' cannot be overloaded with 'void SQLite3::Statement::bind(int, int)'
[   78s]   107 |         void bind(int pos, std::uint32_t val)
[   78s]       |              ^~~~
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/sack/../transaction/../utils/sqlite3/Sqlite3.hpp:93:14: note: previous declaration 'void SQLite3::Statement::bind(int, int)'
[   78s]    93 |         void bind(int pos, int val)
[   78s]       |              ^~~~
[   78s] In file included from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/repo/../conf/ConfigMain.hpp:29,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/repo/../conf/ConfigRepo.hpp:26,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/repo/Repo.hpp:26,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/repo/Repo-private.hpp:24,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/hy-repo-private.hpp:24,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/repo/solvable/Package.hpp:10,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/module/ModulePackage.hpp:30,
[   78s]                  from /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/dnf-sack-private.hpp:32:
[   78s] /home/abuild/rpmbuild/BUILD/libdnf-0.67.0/libdnf/repo/../conf/OptionNumber.hpp:94:41: error: 'int32_t' is not a member of 'std'; did you mean 'int32_t'?
[   78s]    94 | extern template class OptionNumber<std::int32_t>;
[   78s]       |                                         ^~~~~~~
```

due to:
https://gcc.gnu.org/gcc-13/porting_to.html#header-dep-changes

--- a/libdnf/conf/ConfigMain.hpp
+++ b/libdnf/conf/ConfigMain.hpp
@@ -32,6 +32,7 @@
 #include "OptionString.hpp"
 #include "OptionStringList.hpp"
 
+#include <cstdint>
 #include <memory>
 
 namespace libdnf {

--- a/libdnf/conf/ConfigRepo.hpp
+++ b/libdnf/conf/ConfigRepo.hpp
@@ -26,6 +26,7 @@
 #include "ConfigMain.hpp"
 #include "OptionChild.hpp"
 
+#include <cstdint>
 #include <memory>
 
 namespace libdnf {

--- a/libdnf/conf/OptionSeconds.hpp
+++ b/libdnf/conf/OptionSeconds.hpp
@@ -25,6 +25,8 @@
 
 #include "OptionNumber.hpp"
 
+#include <cstdint>
+
 namespace libdnf {
 
 /**

